### PR TITLE
feat: count size update

### PR DESCRIPTION
**Pull request by AmandaKr**
## Jira issue

JIRA issue: [FSU-160](https://klau.atlassian.net/browse/FSU-160)

## Describe your changes

I added 4 sizes available for `Count` component which support 3 breakpoints: desktop, tabletLg and mobileLg.

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/design-system/components/count/Count.jsx [hunk 1]
```diff
@@ -14,11 +14,13 @@ Count.propTypes = {
   count: number.isRequired,
   className: string,
   variant: oneOf(["title", "decorative"]),
```
Comment (im-klau) [inline packages/design-system/components/count/Count.jsx:16]: I don't think you need the `variant` prop anymore 🤔 as it was responsible for changing the size before

Comment (im-klau) [reply packages/design-system/components/count/Count.jsx:16]: also, the DecorativeCard component should be probably update according to the new sizing :D

File: packages/design-system/patterns/decorativeCard/DecorativeCard.jsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ import Typography from "../../components/typography/Typography"
 const DecorativeCard = ({ title, illustrationName, count }) => (
   <Styled.DecorativeCard>
     <Styled.Content>
-      <Count count={count} variant="decorative" />
+      <Count count={count} size="medium" />
```
Comment (im-klau) [inline packages/design-system/patterns/decorativeCard/DecorativeCard.jsx:12]: this is the default size 🙈

**Review by im-klau — APPROVED** [review]
🚀

File: packages/design-system/components/count/Count.styled.js [hunk 3]
```diff
@@ -1,23 +1,44 @@
...
+  }
   text-align: center;
   display: inline-flex;
-  align-items: ${({ variant }) =>
-    variant === "title" ? `center` : `flex-end`};
   font-weight: 900;
-  font-size: ${({ variant }) => (variant === "title" ? `48px` : `110px`)};
-  line-height: 81px;
+  line-height: 74%;
   font-family: "Poppins";
```
Comment (im-klau) [inline packages/design-system/components/count/Count.styled.js:39]: @AmandaKr it's not related to this PR but could you maybe update the font family? 🙏 It should be Montserrat, not Poppins

Comment (AmandaKr) [reply packages/design-system/components/count/Count.styled.js:39]: Sure :D

**Review by paulgrym — APPROVED** [review]
👏

**Pull request merged**